### PR TITLE
feat(sdk)!: allow passing Activity for passive sitekeys

### PR DESCRIPTION
**Pull request by CAMOBAP**
Related:
 - https://github.com/hCaptcha/hcaptcha-android-sdk/issues/112

### CODE REVIEW

**Comment by CAMOBAP** [comment]
@DSergiu for some reason cannot add you as reviewer, could you please have a look also?

**Review by e271828- — APPROVED** [review]
I think this may require a major version bump.

**Review by e271828- — COMMENTED** [review]
Also, please update docs.

**Comment by CAMOBAP** [comment]
> I think this may require a major version bump.

Yep, I think the same

> Also, please update docs.

On it

File: README.md [hunk 1]
```diff
@@ -116,6 +115,8 @@ To remove a specific listener you may use `HCaptcha.removeOn[Success|Failure|Ope
 
 To remove all listeners you may use `HCaptcha.removeAllListener()`.
 
+Note: For a regular use-case (Visual Challenge validation), `HCaptcha.getClient(Activity)` must be called with `FragmentActivity` instance
```
Comment (DSergiu) [inline README.md:118]:
```suggestion
Note ⚠️: For the regular use-case (visual challenge validation), `HCaptcha.getClient(Activity)` must be called with `FragmentActivity` instance. `Activity` is allowed only when `hideDialog=true`.

```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 4.0.0
+
+- Feature: accept `HCaptcha.getClient(Activity)` for passive `siteKey`s (#112)
```
Comment (DSergiu) [inline CHANGES.md:5]:
```suggestion
- Feat (breaking change): accept `HCaptcha.getClient(Activity)` for passive sitekeys. (#112)
```

**Review by DSergiu — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -115,7 +115,8 @@ To remove a specific listener you may use `HCaptcha.removeOn[Success|Failure|Ope
 
 To remove all listeners you may use `HCaptcha.removeAllListener()`.
 
-Note: For a regular use-case (Visual Challenge validation), `HCaptcha.getClient(Activity)` must be called with `FragmentActivity` instance
+Note ⚠️: For the regular use-case (visual challenge validation), `HCaptcha.getClient(Activity)` must be called with `FragmentActivity` instance. `Activity` is allowed only when `hideDialog=true`.
```
Comment (e271828-) [inline README.md:118]:
```suggestion
Note ⚠️: For any sitekey that can show visual challenges, `HCaptcha.getClient(Activity)` must be called with `FragmentActivity` instance.

`Activity` is allowed only when `hideDialog=true` and sitekey setting is Passive (Enterprise feature).
```

**Comment by CAMOBAP** [comment]
It starts hanging quite often, looks like a known issue https://github.com/ReactiveCircus/android-emulator-runner/issues/381

**Pull request merged**